python/ray/rllib/dqn/models.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import division
 from __future__ import print_function
 
+import numpy as np
```

Comment:
why import?

Reply:
This is just re-ordering them so third-party imports are grouped

Reply:
oh this is a lint error